bazaarvoice/emodb
# Consolidate system table placement

## Github Issue #

[53](https://github.com/bazaarvoice/emodb/issues/53)

## What Are We Doing Here?

This PR consolidates the system table placement into the upper level emo configuration and removes it from subset configurations. The emo guice module now injects the placement into the dependent modules.

## How to Test and Verify

1. Check out this PR
2. Build Emo
3. Run locally
4. Run `cqlsh` and observe that app_global:sys is still being used as the system table

## Risk

### Level 

`Low`

### Required Testing

`Smoke`

### Risk Summary

This is extremely low risk, as any potential issue would be quickly discovered on boot.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
